Steam Web Integration.user.js

```diff
@@ -55,6 +55,7 @@ function displaySettings() {
     $("#title").text(`${name} (${version}) by ${author}`);
     $("#settings").show();
     $("#notinstalled").hide();
+    $("#blackList").attr("placeholder", "example1.com" + "\n" + "domain2" + "\n" + "/bundle/");
```

Comment:
why not put this in the html page directly?

Reply:
- Doesn't have a script tag; this does.
- Doesn't have jQuery; this does.
- Why this is even JS: I guess I misplaced the commit message. https://stackoverflow.com/questions/7312623/insert-line-break-inside-placeholder-attribute-of-a-textarea — a few browsers are happy with `&#10;` but others are really not.